brd-ios/breadwallet/src/Redesign/Views/Core/FETextField.swift

```diff
@@ -34,9 +34,10 @@ struct TextFieldConfiguration: Configurable {
 }
 
 extension TextFieldConfiguration {
-    mutating func setSecure(_ isSecure: Bool) -> TextFieldConfiguration {
-        isSecureTextEntry = isSecure
-        return self
+    func setSecure(_ isSecure: Bool) -> TextFieldConfiguration {
+        var copy = self
+        copy.isSecureTextEntry = isSecure
+        return copy
```

Comment:
interesting

Reply:
man.. i was baffled... will def discuss this on the weekly

Reply:
you were proud of mutable funcs :D